bindings/nodejs/lib/types/models/api/mana-rewards-response.ts

```diff
@@ -0,0 +1,31 @@
+// Copyright 2024 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import { EpochIndex } from '../../block/slot';
+import { u64 } from '../../utils';
+
+/**
+ * Returns the mana rewards of an account or delegation output.
```

Comment:
There is more of these `Returns` comments

Reply:
All from the TIP :runner:

Reply:
Updated